packages/amplify-category-auth/resources/adminAuth/admin-auth-app.js

```diff
@@ -178,14 +175,11 @@ app.get('/listUsers', async (req, res, next) => {
 
 app.get('/listGroups', async (req, res, next) => {
   try {
-    let response;
-    if (req.query.token) {
-      response = await listGroups(req.query.limit || 25, req.query.token);
-    } else if (req.query.limit) {
-      response = await listGroups((Limit = req.query.limit));
-    } else {
-      response = await listGroups();
+    let limit = 25;
+    if (req.query.limit) {
+      limit = parseInt(req.query.limit);
     }
```

Comment:
nit: Would is be _slightly_ easier to read?
```suggestion
      const limit = req.query.limit ? parseInt(req.query.limit) : 25;
```

 (and the rest)